app/controllers/api/v1/passwords_controller.rb

```diff
@@ -3,15 +3,18 @@
 module API
   module V1
     class PasswordsController < DeviseTokenAuth::PasswordsController
-      protect_from_forgery with: :exception, unless: :json_request?
       include API::Concerns::ActAsAPIRequest
-      skip_before_action :check_json_request, on: :edit
```

Comment:
Is this being removed because the method is not used in an api or because of some other reason?

Reply:
exactly, the `edit` path is not even accessible. Not sure why this was here in first place

Reply:
FTR seems like the edit path is used to redirect to the reset password in the frontend app